includes/Modules/Thank_With_Google.php

```diff
@@ -314,7 +314,12 @@ function ( $host ) {
 						}
 					}
 
-					if ( count( $sidebars ) === $actual_sidebars_count ) {
+					if (
+						// Only show the "All sidebars" text if there is more
+						// than two sidebars.
+						$actual_sidebars_count >= 2 &&
```

Comment:
This looks like it will work but the comment doesn't quite match the code as it says "more than two" which should really be "more than one".

Then I think it would make sense to tweak the condition to match, which is also slightly simpler.
```suggestion
						// Only show the "All sidebars" text if there is more
						// than one sidebar.
						$actual_sidebars_count > 1 &&
```

Reply:
Hah, whoops, thanks 😄